package.json

```diff
@@ -33,6 +33,7 @@
     "@tauri-apps/api": "^1.5.0",
     "@tauri-apps/cli": "^1.5.1",
     "@types/node": "^20.8.2",
+    "@types/w3c-web-serial": "^1.0.6",
```

Comment:
Actually, this is an artifact, can be removed :)

Reply:
Sure, I'll remove it :)